plugins/block-shareable-procedures/src/blocks.ts

```diff
@@ -450,20 +454,19 @@ const procedureDefMutator = {
    */
   loadExtraState: function(state) {
     const map = this.workspace.getProcedureMap();
+
     const procedureId = state['procedureId'];
-    if (procedureId && procedureId != this.model_.getId() &&
-      map.has(procedureId) &&
-      (this.isInsertionMarker() ||
```

Comment:
Why do you no longer have to check whether it's an insertion marker?

Reply:
Insertion markers are saved with `doFullSerialization: false` so it is covered by the other check.